jazzmin/templates/admin/change_list.html

```diff
@@ -111,5 +111,14 @@ <h4 class="card-title">{{ title }}{% block pretitle %}{% endblock %}</h4>
 {% block extrajs %}
     {{  block.super }}
     <script type="text/javascript" src="{% static 'vendor/select2/js/select2.min.js' %}"></script>
+    <script>
+        {% comment %} set filterInputLength default and custom values {% endcomment %}
+        window.filterInputLengthDefault = 0;
+        window.filterInputLength = {
```

Comment:
Is there value in being able to set the minimum input length per model admin ? Or would it simplify things to have it set within `settings.JAZZMIN_SETTINGS` ?,

Reply:
> It's per admin model. Some filters may have 20K+ records and some just 5. I think we can use global setting to provide a default, and then customize it if needed